tests/smoke_tests/run_smoke_test.py

```diff
@@ -38,7 +46,10 @@ async def run_smoke_test(
     # reads lines from the server output in search of the startup log message
     # times out after 20s of inactivity if it doesn't find the log message
     full_server_output = ""
-    startup_message = "FL starting"
+    startup_messages = [
+        "FL starting",
+        "Using Warm Start Strategy. Waiting for clients to be available for polling",
```

Comment:
Maybe mention why there are two messages here. That is, the final message is different depending on the server (SCAFFOLD prints the second)